src/main/java/seedu/duke/commands/ExitCommand.java

```diff
@@ -1,12 +1,17 @@
 package seedu.duke.commands;
 
+import java.util.logging.Logger;
+
 public class ExitCommand extends Command {
+    private static final Logger LOGGER = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
```

Comment:
Looking at https://github.com/se-edu/addressbook-level3/blob/master/src/main/java/seedu/address/logic/LogicManager.java, I think it is completely fine to have the `Logger` at the main `Command` class. It's just that we haven't thought of logging for the other commands, but I think this should be moved to the main class.

Reply:
Okay fair, I'll move it